libs/service-portal/documents/src/screens/Overview/Overview.tsx

```diff
@@ -155,6 +155,33 @@ export const ServicePortalDocuments: ServicePortalModuleComponent = ({
 
   const hasActiveFilters = () => !isEqual(filterValue, defaultFilterValues)
 
+  const documentsFoundText = () => {
+    // default text format, singular & plural
+    let foundText =
+      filteredDocuments.length === 1
+        ? formatMessage({
```

Comment:
formatMessage should only be used when rendering, you can use defineMessage when declaring variables and then format it in the markup